src/Symfony/Component/VarDumper/Test/VarDumperTestTrait.php

```diff
@@ -19,6 +19,26 @@
  */
 trait VarDumperTestTrait
 {
+    /** @internal */
+    private static $casters = [];
+    /** @internal */
+    private static $flags;
```

Comment:
we should give these properties a more specific name, to never collide with any other userland property
what about putting them all in one single property, to reduce polluting the scope?

Reply:
would it hurt to just initialize the cloner in `setUpVarDumper`... that's a setup no :)